src/dmd/backend/dwarfdbginf.d

```diff
@@ -1133,7 +1133,7 @@ version (MARS)
 {
     debug_info.buf.write("Digital Mars D ");
     //debug_info.buf.writeString(global._version);     // DW_AT_producer
-    debug_info.buf.writeString("v2.083");              // DW_AT_producer
+    debug_info.buf.writeString("v2.084");              // DW_AT_producer
```

Comment:
should fix it in elfobj.d, too.

Reply:
Should this be 2.085? 2.084 has already been released.

Reply:
Is there no plan for point release?

Reply:
If you want it in a point release, you should target stable